README.md

```diff
@@ -196,6 +196,13 @@ next_url = page1.next_page_url
 page2 = shopify.Product.find(from_=next_url)
 ```
 
+## Development on this repo
+Install requirements and initialize [pre-commit](https://pre-commit.com/)
```

Comment:
I wonder if this could be seen as a barrier to contribution for the repo. Maybe it could be marked as optional?

Reply:
(I see there's a [GitHub Action](https://github.com/pre-commit/action) that could be used to ensure the checks are still performed).

Reply:
I was thinking about that too, we've usually preferred to have CI actions for this kind of thing to reduce friction. I'd prefer to use actions if they're an option.

Reply:
okay fixed it! Thank you